src/util/color/rgbcolor.h

```diff
@@ -135,6 +135,23 @@ inline QVariant toQVariant(RgbColor::optional_t optional) {
     }
 }
 
+// Explicit conversion of both non-optional and optional
+// RgbColor values to QString with the format #RRGGBB,
+// e.g. for tool.
+
+inline QString toQString(mixxx::RgbColor color) {
+    return mixxx::toQColor(color).name();
+}
+
+// String representation #RRGGBB, e.g. for tool.
+inline QString toQString(mixxx::RgbColor::optional_t color) {
```

Comment:
Maybe add a optional argument that allows influencing the output if no color is set? For the track color picker I could set this to "None" or "Transparent".